[2023-10-14T02:50:58.261+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dataframe_lab1.change_datatype scheduled__2023-10-08T00:00:00+00:00 [queued]>
[2023-10-14T02:50:58.286+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dataframe_lab1.change_datatype scheduled__2023-10-08T00:00:00+00:00 [queued]>
[2023-10-14T02:50:58.289+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-14T02:50:58.328+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): change_datatype> on 2023-10-08 00:00:00+00:00
[2023-10-14T02:50:58.345+0000] {standard_task_runner.py:57} INFO - Started process 112 to run task
[2023-10-14T02:50:58.356+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dataframe_lab1', 'change_datatype', 'scheduled__2023-10-08T00:00:00+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/titanic.py', '--cfg-path', '/tmp/tmppv9eaz47']
[2023-10-14T02:50:58.365+0000] {standard_task_runner.py:85} INFO - Job 97: Subtask change_datatype
[2023-10-14T02:50:58.478+0000] {task_command.py:415} INFO - Running <TaskInstance: dataframe_lab1.change_datatype scheduled__2023-10-08T00:00:00+00:00 [running]> on host 89ffa62682cc
[2023-10-14T02:50:59.223+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dataframe_lab1' AIRFLOW_CTX_TASK_ID='change_datatype' AIRFLOW_CTX_EXECUTION_DATE='2023-10-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-08T00:00:00+00:00'
[2023-10-14T02:50:59.296+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-14T02:50:59.317+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=dataframe_lab1, task_id=change_datatype, execution_date=20231008T000000, start_date=20231014T025058, end_date=20231014T025059
[2023-10-14T02:50:59.377+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-14T02:50:59.406+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
